examples/angular_bazel_architect/BUILD.bazel

```diff
@@ -112,6 +112,26 @@ architect_test(
     ],
 )
 
+architect(
+    name = "lint",
+    args = ["frontend:lint"],
+    data = glob([
+        "src/**/*",
```

Comment:
From https://docs.bazel.build/versions/master/be/functions.html#glob

> /bar//*.txt matches every .txt file in every subdirectory of this package, if at least one directory on the resulting path is called bar, such as xxx/bar/yyy/zzz/a.txt or bar/a.txt (remember that ** also matches zero segments) or bar/zzz/a.txt

Reply:
Hmm! In copybara, which also uses starlark for configuration, their glob implementation has different semantics!
https://github.com/google/copybara/blob/master/docs/reference.md#matching-multiple-strings-with-one-expression
so #TIL that "glob" isn't a starlark builtin and things are not consistent